### ISSUE TITLE
Overwriting basetype bug

### ISSUE THREAD

**Issue by bannanc** [open-forcefield-group/smarty#54]
```
        # Store a copy of the basetypes, as these (and only these) are allowed
        # to end up with zero occupancy
        self.basetypes = copy.deepcopy(self.atomtypes)
```

We are reading in a basetype file and then over writing this list with the initial atom types which are read into `self.atomtypes`

I will fix this shortly, but I thought I would point it out. 

I've also been thinking that since we're treating basetypes specially (and generally assuming they are elements) maybe they should be hard coded somewhere and not fed in from a file.

**Comment by bannanc** [comment]
@davidlmobley , @camizanette  and @cbayly13 thoughts?

**Comment by davidlmobley** [comment]
@bannanc - this must have gotten added back in on a merge (probably Camilla's changes), as I'd removed this when I was implementing the possibility of having separate base and initial atom types. Please fix. :) 

Thanks.

**Comment by bannanc** [comment]
I've included it in the pull request just submitted, also though that has a number of other concerns.

**Comment by davidlmobley** [comment]
Resolved in #55.
